python/activity_stream/metric_utils.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2017 Shotgun Software Inc.
```

Comment:
So in the other PRs, the property gathering and metric logging was inline with the rest of the code. Here you've moved the actual logging to a separate util file. Suggest we just do this inline to be consistent unless there's a strong reason not to.

Reply:
The reason was that I needed to emit the same metric from different modules (2 for now) maybe more is the future?